package.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "laravel-vite-plugin",
+    "type": "module",
```

Comment:
This is required to use MJS in the purge script.

Reply:
You can also simply write the binary in TypeScript and use a build tool to transpile it back to CJS.

- [`unbuild`](https://github.com/unjs/unbuild) is great, I use it in a lot of projects. It's built on top of Rollup.
- [`tsup`](https://github.com/egoist/tsup) is also a good alternative. It's built on top of esbuild.

I personally recommend `unbuild` and I can help you setting that up if needed 👍

Reply:
We'll be making this change as part of the `1.x` release anyway, so I think it's good with this.